### PR TITLE
Add janus requests timeout

### PR DESCRIPTION
**Pull request by feymartynov**

### CODE REVIEW

File: src/app/janus.rs [hunk 1]
```diff
@@ -34,6 +37,135 @@ use crate::util::{from_base64, generate_correlation_data, to_base64};
 
 const STREAM_UPLOAD_METHOD: &str = "stream.upload";
 
+lazy_static! {
```
Comment (manifest) [inline src/app/janus.rs:40]: Let's move these parameters to the app configuration. `[backend]` section for example.

Comment (manifest) [reply src/app/janus.rs:40]: `[backend]` section would also be a nice place for `backend_id`, meaning

```
[backend]
id = "janus-gateway.svc.example.org"
```

Comment (manifest) [reply src/app/janus.rs:40]: `src/backend` would probably be a better place for this entire file.

Comment (feymartynov) [reply src/app/janus.rs]: Done. Also refactored it into smaller modules.

**Review by manifest — APPROVED** [review]

**Review by khodzha — APPROVED** [review]

File: src/app/endpoint/rtc_signal.rs [hunk 2]
```diff
@@ -74,20 +73,21 @@ impl RequestHandler for CreateHandler {
...
-                    .status(ResponseStatus::UNPROCESSABLE_ENTITY)?
+                    context
+                        .janus_client()
+                        .read_stream_request(
+                            reqp.clone(),
+                            payload.handle_id.janus_session_id(),
+                            payload.handle_id.janus_handle_id(),
+                            payload.handle_id.rtc_id(),
+                            payload.jsep.clone(),
+                            payload.handle_id.backend_id(),
```
[the remark this answers is not in the document]
Comment (feymartynov) [reply src/app/endpoint/rtc_signal.rs:84]: No, it's not OK. There's a lot of room for refactoring. I've just moved code as is for now.

File: src/app/endpoint/rtc_signal.rs [hunk 3]
```diff
@@ -74,20 +73,21 @@ impl RequestHandler for CreateHandler {
...
+                            start_timestamp,
+                            authz_time,
+                        )
+                        .map(|req| Box::new(req) as Box<dyn IntoPublishableMessage + Send>)
+                        .map_err(|err| format!("error creating a backend request: {}", err))
```
[the remark this answers is not in the document]
Comment (feymartynov) [reply src/app/endpoint/rtc_signal.rs:89]: Same about error. There's another task for improving them.

**Pull request merged**